### ISSUE TITLE
UI/UX Testing 🔨 

### ISSUE THREAD

**Issue by MatthewSzurkowski** [GBADsInformatics/GBADsDataSearchUI#9]
This is an issue that can be addressed once the site has reached near MVP completion.

**Comment by kassyray** [comment]
Kassy needs to try out a bunch of queries and just play with the UI in general

**Comment by MatthewSzurkowski** [comment]
Issue with scroll on main search screen. Need to update the body scroll to be toggled once a search has been entered...

**Comment by kassyray** [comment]
Do you still need me to do this? I totally forgot.

**Comment by MatthewSzurkowski** [comment]
@kassyray Yes I'll be testing myself as well. If you notice any issues, please raise them here and I can resolve them :)

**Comment by MatthewSzurkowski** [comment]
Pushed 84b71af which contains an automated test program for the UI. Performs UI validation on the keywords.
